src/clx.js

```diff
@@ -0,0 +1,35 @@
+import { cache } from './core/hash';
```

Comment:
If we want to keep `clx` out of the core, this cache will need to be exported from `goober` itself. Currently it's only available from the `core/hash` module directly.